src/njoy21/input/ACER/Card7/test/Card7.test.cpp

```diff
@@ -0,0 +1,57 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "Verifying ACER Card7 input", 
+         "[ACER], [Card7]" ){
+  GIVEN( "all three entries" ){
+    THEN( "the appropriate values are returned" ){
+      iRecordStream<char> issCard7( std::istringstream( "-2.0 1E3 1/" ) );
```

Comment:
doesn't make sense. you've asked for integral fraction thinning, but are providing a iskf argument